### DISCUSSION TITLE
Transferring a website to a vvveb cms from one domain to another with an Sqlite database.

### DISCUSSION THREAD

**Discussion by aukc1970** [givanz/Vvveb#234] [General]
Good afternoon! Please tell me, when transferring a site on your cms from one domain to another with an Sqlite database, do you need to fix anything in the database as well as in the settings of the cms itself?

**Comment by givanz** [comment]
Hi

It is enough to copy the sqlite database file `/storage/sqlite/vvveb.db` no additional changes are necessary.
Urls  and other settings are not saved with the domain name included so no search and replace is needed.
If you need to transfer multi site or active plugins settings you also need to copy `/config` folder.

**Reply by aukc1970** [reply]
Thank you very much!
